### DISCUSSION TITLE
Smartbanner not displaying on chrome and safari

### DISCUSSION THREAD

**Discussion by noontinyy** [ain/smartbanner.js#723] [Q&A] [answered]
Hello. I do have a problem using smartbannerjs on nextjs.
It's not displaying on any browser and I'm not sure why.

Here's my code

`<meta name="smartbanner:title" content="Smart Application" />
        <meta name="smartbanner:author" content="SmartBanner Contributors" />
        <meta name="smartbanner:price" content="FREE" />
        <meta
          name="smartbanner:price-suffix-apple"
          content=" - On the App Store"
        />
        <meta
          name="smartbanner:price-suffix-google"
          content=" - In Google Play"
        />
        <meta
          name="smartbanner:icon-apple"
          content="url-icon"
        />
        <meta
          name="smartbanner:icon-google"
          content="url-icon"
        />
        <meta name="smartbanner:button" content="VIEW" />
        <meta
          name="smartbanner:button-url-apple"
          content="url-apple-app"
        />
        <meta
          name="smartbanner:button-url-google"
          content="url-google-app"
        />
        <meta name="smartbanner:enabled-platforms" content="android,ios" />
        <meta name="smartbanner:close-label" content="Close" />`
        
Did I import css and js correctly??
<img width="281" alt="image" src="https://github.com/ain/smartbanner.js/assets/73950414/8d05b752-d2b1-4294-b7a8-011adb91d34e">
<img width="559" alt="image" src="https://github.com/ain/smartbanner.js/assets/73950414/e9454b4c-33e9-4daa-bd79-a75c7597ab45">

Also I don't have any error on my console.

**Comment by ain** [comment, marked as answer]
I'd advise to avoid any effects and rely on the [Basic usage](https://github.com/ain/smartbanner.js?tab=readme-ov-file#basic-usage) when loading JavaScript and CSS files as well as use [API](https://github.com/ain/smartbanner.js?tab=readme-ov-file#smartbanner-api-use) to work with the smartbanner.js.

You can also use absolute URLs to ensure smartbanner.js CSS/JS is loaded correctly.

**Reply by noontinyy** [reply]
Hi! Thank you 😊 
It's showing perfectly now!

**Comment by omerzirh** [comment]
@noontinyy is it working for you in chrome with iOS device?
